src/components/mediaManager/AposMediaManager.vue

```diff
@@ -160,6 +161,21 @@ export default {
     },
 
     // select setters
+    toggle(id) {
+      // Toggling using the checkbox element.
+      // REVIEW: Should this simply use part of `selectAnother` to deselect?
```

Comment:
I think adding this method is confusing but more because `selectAnother` is ACTUALLY the toggler method .. if you could roll them together into `selectToggle` and make them both leverage it I think that would be clearer.
(the `selectToggle` name only suggested because of the name spacing of the select setters)